<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1039</width>
    <height>675</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>632</width>
      <height>455</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="VvodLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Ввод данных</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="EllipsView" name="ellipsView" native="true">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>630</width>
         <height>430</height>
        </size>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>20</y>
      <width>321</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="GraphLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Вывод графа</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="GraphView" name="graphView" native="true"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>320</y>
      <width>321</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="PowerLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Вывод мощности</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="PowerView">
       <property name="textInteractionFlags">
        <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>600</y>
      <width>201</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="RallButton">
       <property name="text">
        <string>Reset All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ComputationButton">
       <property name="text">
        <string>Computation</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>490</y>
      <width>301</width>
      <height>159</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="Emptylabel">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="CenterLabel">
           <property name="text">
            <string>Центр</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="WidthLabel">
           <property name="text">
            <string>Ширина</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="HeightLabel">
           <property name="text">
            <string>Высота</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="1" column="1">
            <widget class="QLineEdit" name="YEdit"/>
           </item>
           <item row="0" column="0" alignment="Qt::AlignHCenter">
            <widget class="QLabel" name="XLabel">
             <property name="text">
              <string>x</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" alignment="Qt::AlignHCenter">
            <widget class="QLabel" name="YLabel">
             <property name="text">
              <string>y</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="XEdit">
             <property name="inputMethodHints">
              <set>Qt::ImhNone</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLineEdit" name="WidthEdit"/>
         </item>
         <item>
          <widget class="QLineEdit" name="HeightEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="ResetButton">
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="AddButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>&amp;Add</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="EllipsTable">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>490</y>
      <width>321</width>
      <height>161</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>76</number>
    </attribute>
    <column>
     <property name="text">
      <string>X</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Y</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Height</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Widht</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EllipsView</class>
   <extends>QWidget</extends>
   <header>ellipsview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GraphView</class>
   <extends>QWidget</extends>
   <header>graphview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>XEdit</tabstop>
  <tabstop>YEdit</tabstop>
  <tabstop>WidthEdit</tabstop>
  <tabstop>HeightEdit</tabstop>
  <tabstop>AddButton</tabstop>
  <tabstop>ResetButton</tabstop>
  <tabstop>PowerView</tabstop>
  <tabstop>EllipsTable</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>ResetButton</sender>
   <signal>clicked()</signal>
   <receiver>XEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>630</y>
    </hint>
    <hint type="destinationlabel">
     <x>117</x>
     <y>530</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ResetButton</sender>
   <signal>clicked()</signal>
   <receiver>WidthEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>639</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>569</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ResetButton</sender>
   <signal>clicked()</signal>
   <receiver>HeightEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>629</y>
    </hint>
    <hint type="destinationlabel">
     <x>118</x>
     <y>595</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ResetButton</sender>
   <signal>clicked()</signal>
   <receiver>YEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>53</x>
     <y>637</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>542</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
